src/main/java/org/rsa/command/subcommands/configure/ReactionsSubcommand.java

```diff
@@ -0,0 +1,42 @@
+package org.rsa.command.subcommands.configure;
+
+import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent;
+import net.dv8tion.jda.api.interactions.commands.OptionType;
+import net.dv8tion.jda.api.interactions.commands.build.OptionData;
+import org.rsa.command.SubcommandObject;
+import org.rsa.logic.data.managers.GuildConfigurationManager;
+import org.rsa.logic.data.models.GuildConfiguration;
+
+import java.util.Objects;
+
+public class ReactionsSubcommand extends SubcommandObject {
+    public ReactionsSubcommand()
+    {
+        super("reactions", "Change reaction emojis");
+        this.addOptions(new OptionData(OptionType.STRING, "reaction_type", "Specify the reaction emoji to change.", true)
+                .addChoice("upvote", "upvote")
+                .addChoice("downvote", "downvote"),
```

Comment:
Can we move `new OptionData(...)` to a new line, then tab the `.addChoice(...)`?